scripts/pack_server.sh

```diff
@@ -121,14 +114,20 @@ copy_file ./rdsn/thirdparty/output/lib/libk5crypto.so.3 ${pack}/bin
 copy_file ./rdsn/thirdparty/output/lib/sasl2 ${pack}/bin
 copy_file ./scripts/sendmail.sh ${pack}/bin
 copy_file ./src/server/config.ini ${pack}/bin
+copy_file ./src/server/config.min.ini ${pack}/bin
```

Comment:
Why adding pack `config.min.ini` here?

Reply:
For testing / low-resources cases. The server package could be used by docker. I think config.ini is for the production environment that is to maximize performance, config.min.ini is to minimize resource usage. Both configs have their use-cases.